### ISSUE TITLE
Use Indeterminate checkboxes in RDNS Plus Simple view

### ISSUE THREAD

**Issue by crisp5** [celzero/rethink-app#703]
When configuring RDNS Plus blocklists, there are two views: Simple (groups of blocklists) and Advanced (list of all blocklists). If I make a selection in the Simple View, but remove one of its blocklists in the Advanced view, returning to the Simple view shows that group unselected. I think it would be more intuitive to use indeterminate checkboxes (box with a dash in it) when we use a subset of the blocklists in a certain group. Thank you!

Version: v053l (play)

**Comment by ignoramous** [comment]
Makes sense. Thanks. A bit tougher to implement (given the ridig assumptions in the code base), but I think it is worth prioritizing this.

**Comment by ignoramous** [comment]
Indeterminate checkboxes aren't part of Android's standard widget toolkit... This means, we'd have to impl our own widget (or use an open source one of which there are a handful but quality / maintenance aspects are unclear)... not straight-forward to do, at all; and so, it might be a while before we address this.

**Comment by crisp5** [comment]
That's understandable, thanks. My main concern was not having an indication of whether I'm using every list in a group or not, which can change from update to update as lists are updated/added/removed.
An alternate solution could be a button similar to the "1234 entries" button each blocklist has in Advanced mode, but instead saying something like "5/6 lists selected". Further, pressing it could show a modal listing the lists in the group, like the "+1 Other IPs" button in the Logs view. 
I'll leave it up to you to decide whether either path is worth following
